pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1888,12 +1889,14 @@ static __always_inline bool context_changed(task_context_t *old, task_context_t
            *(u64 *) &old->uts_name[8] != *(u64 *) &new->uts_name[8];
 }
 
-static __always_inline int init_program_data(program_data_t *p, void *ctx)
+// clang-format off
+static __always_inline int init_program_data(program_data_t *p, void *ctx, bool kthread)
```

Comment:
Instead of adding a new argument to this function and changing all references, you can add a new field to program_data_t which will tell the buffer index to use